python/lsst/verify/gen2tasks/metricRegistry.py

```diff
@@ -22,38 +22,39 @@
 __all__ = ["register", "registerMultiple"]
 
 from lsst.pex.config import Config, ConfigDictField, Registry
-from .metricTask import MetricTask
+from ..tasks.metricTask import MetricTask
```

Comment:
Actually, no, I can't. Using an absolute import causes circular dependency problems (`lsst.verify.tasks` must depend on `lsst.verify.gen2tasks` because it uses the registry code). I thought I added a comment about this, but it seems to be gone.